deployments/porch/5-rbac.yaml

```diff
@@ -68,3 +68,6 @@ rules:
   - apiGroups: [""]
     resources: ["pods"]
     verbs: ["create", "delete", "patch", "get", "watch", "list"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "create", "delete"]
```

Comment:
Extra-nitpicky - consider moving the `get` to the end of the list, just to have them in the same order as for the pods entry above

Reply:
changed order to match and added patch for use in patching the pod secret if a mismatch with the user edited secret occurs.

Reply:
Looks good - OK to resolve